articles/azure/azs-how-create-vmfromsnapshot-portal.md

```diff
@@ -0,0 +1,147 @@
+    - **Subscription** - This will be your UKCloud for Microsoft Azure subscription.
+
+    - **Resource group** - Select an existing resource group, or create a new one by typing a name for your new resource group.
+
+    - **Location** - This will be the location of the Azure Stack Hub.
+
+    - **Account type** - Choose either Standard (HDD) or Premium (SSD) storage.
+
+    ![Create snapshot from disk - Populate & Create](images/azs-browser-create-snapshot.png)
+
+6. When you're done, click the **Create** button to create the snapshot.
+
+7. You can monitor the progress of the snapshot creation by clicking the **Notifications** icon.
+
+    ![Notification showing snapshot creation in progress](images/azs-browser-create-snapshot-progress.png)
+
+## Creating a new managed disk from a snapshot
+
+1. Click **All services** in the favourites panel, then select **Disks** under the *Compute* section.
+
+    ![All services - disks](images/azs-browser-allservices-disks.png)
+
+2. On the **Disks** blade, click **Add**.
+
+    ![Disks blade - Add](images/azs-browser-disks-list-add.png)
+
+3. In the **Create managed disk** blade, enter the following information:
+
+    - **Name** - The name of the disk.
+
+    - **Subscription** - This will be your UKCloud for Microsoft Azure subscription.
+
+    - **Resource group** - Select an existing resource group, or create a new one by typing a name for your new resource group.
+
+    - **Location** - This will be the location of the Azure Stack Hub.
+
+    - **Account type** - Choose either Standard (HDD) or Premium (SSD) storage.
+
+    - **Source type** - Select *Snapshot* from the dropdown.
+        - **Source snapshot** - Select the previously created snapshot.
+
+    - **Size** - This will be set to the size of the source disk that the snapshot was created from. You can increase this if necessary, but you will not be allowed to decrease it below the original size.
+
+    ![Create disk from snapshot](images/azs-browser-create-disk-from-snapshot.png)
+
+4. When you're done, click the **Create** button to create the disk.
+
+5. You can monitor the progress of the disk creation by clicking the **Notifications** icon.
+
+    ![Notification showing disk creation in progress](images/azs-browser-create-disk-from-snapshot-progress.png)
+
+## Creating a virtual machine from a managed disk
+
+After you have the managed disk that you want to use, you can create the VM in the portal:
+
+1. Click **All services** in the favourites panel, then select **Disks** under the *Compute* section.
+
+    ![All services - disks](images/azs-browser-allservices-disks.png)
+
+2. From the list, select the disk that you would like to create a virtual machine from.
+
+    ![Disks blade](images/azs-browser-disks-list.png)
+
+3. In the new blade for the selected disk, under *Overview*, ensure that **Disk state** is listed as *Unattached*. If it isn't, you will need to either detach the disk from the VM or delete the VM to free up the disk.
```

Comment:
Not sure if deleting the entire VM is necessary. Detaching is fine IMO.

Reply:
Reworded in latest commit.